### PR TITLE
Upgrade ember-one-way-controls for Ember 3 support

### PR DESCRIPTION
**Pull request by danwenzel**
This is required to avoid an error in Ember 3 when using textareas: https://github.com/DockYard/ember-one-way-controls/issues/185.

Also includes the commits from https://github.com/martndemus/ember-form-for/pull/201

### CODE REVIEW

**Comment by amiel** [comment]
> We should probably get tests working before merging this. See #201

It may be the other way around. I think this is required in order to get the tests working :D

**Comment by danwenzel** [comment]
Ah, you're right.  The test failures here look like actual failures, probably related to the upgrade.  I'll have to take a closer look

**Comment by amiel** [comment]
@danwenzel many of the actual failures have been fixed in #201, so don't look too carefully. I'm going to try to merge this PR in to #201 to see if it fixes the last failure.

**Comment by amiel** [comment]
Nevermind, #201 is actually good to go. 

I think I was all confused because I was trying to run it with a newer ember-try configuration (that required beta and canary to pass). The actual configuration allows beta and canary to fail. They still do, and I believe this change will make them pass.

My thought is that #201 should be merged first, and if you rebase on #201, the failures in 3.0 should end up resolved (I hope).

**Comment by danwenzel** [comment]
@amiel - I merged in your branch, and that fixed all the test failures I was seeing - and Ember 3 is now working too.

@martndemus @lcpriest - So we could either merge these separately, or merge this one - which includes both.

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "ember-cli-babel": "^5.1.7",
     "ember-cli-htmlbars": "^1.0.10",
     "ember-invoke-action": "1.4.0",
-    "ember-one-way-controls": "^2.0.0",
+    "ember-one-way-controls": "3.1.0",
```
Comment (musaffa) [inline package.json]: It should be "ember-one-way-controls": "^3.1.0" to incorporate future release fixes.

**Comment by amiel** [comment]
@danwenzel I'm glad to hear it.

File: package.json (hunk 1 above)
Comment (danwenzel) [reply package.json]: Done!

**Comment by danwenzel** [comment]
@martndemus @lcpriest @amiel - Anyone up for merging and releasing this?

**Comment by danwenzel** [comment]
Whoops, meant to tag @musaffa

**Comment by danwenzel** [comment]
cc @marxsk (this one is needed for the addon to work with Ember 3)

**Comment by marxsk** [comment]
@danwenzel I'm aware of importance of this PR but it contains too much different commits. I will start with amiel's PR that you use as well.

**Comment by danwenzel** [comment]
Yup, totally understandable.  Sounds like a good plan to me

**Comment by danwenzel** [comment]
In fact, I'll open a separate, cleaner PR...

**Comment by danwenzel** [comment]
Closing, in favor of https://github.com/martndemus/ember-form-for/pull/204

**Pull request closed without merging**